reading-data/reading_data_solution.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""reading_data
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1YKoHc9bkg7ghdlA_n00X1ckbQ_PSgBjO
+
+# **mtcars.csv**
+"""
+
+import numpy as np
+import pandas as pd
+
+mtcars = pd.read_csv("/content/mtcars.csv")
+mtcars
+
+mtcars_new = mtcars.to_csv("/content/mtcars2.csv") # Choose a new name for the document inside the arugment
+
+mtcars_new_read = pd.read_csv("/content/mtcars2.csv")
+mtcars_new_read
+
+"""# **u.data**"""
+
+u_data = pd.read_csv("/content/u.data", delimiter = '\t', header = None)
+u_data
+
+u_data_2 = u_data.to_csv("/content/u2.data")
+
+u_data_2_read = pd.read_csv("/content/u2.data")
+u_data_2_read
+
+"""# **beer.txt**"""
+
+beer = pd.read_fwf("/content/beer.txt")
+beer
+
+beer_2 = beer.to_csv("/content/beer2.txt")
+
+beer_2_read = pd.read_csv("/content/beer2.txt")
+beer_2_read
+
+"""# **u.item**"""
+
+u_item = pd.read_csv("/content/u.item", delimiter = '|', encoding='ISO-8859-1', header = None)
+u_item
+
+u_item_new = u_item.to_csv("/content/u2.item")
+
+u_item_new_read = pd.read_csv("/content/u2.item")
+u_item_new_read
+
+"""# **NHL 2015-16.xlsx**"""
+
+nhl = pd.read_excel('/content/NHL 2015-16.xlsx')
+nhl
+
+nhl_2 = nhl.to_excel('/content/NHL_2 2015-16.xlsx')
+
+nhl_2_read = pd.read_excel('/content/NHL_2 2015-16.xlsx')
+nhl_2_read
+
+"""# **Function That Reads Without Exceptions**"""
+
+# I'm unsure about how I would approach this with pandas
+# I found suggestions via stack overflow to utilize the csv library
+
+def read_a_file(file_name):
```

Comment:
You should use the same functions to read the files that were used before so that you get a dataframe returned